test/metabase/db/custom_migrations_test.clj

```diff
@@ -1761,7 +1761,7 @@
           (is (= 0 (count (pulse-channel-test/send-pulse-triggers pulse-id)))))
         (testing "the init-send-pulse-triggers job should be re-run after migrate up"
           (migrate!)
-          ;; we need to redef this so quarzt trigger that run on a different thread use the same db connection as this test
+          ;; we need to redef this so quarz trigger that run on a different thread use the same db connection as this test
```

Comment:
```suggestion
          ;; we need to redef this so quartz trigger that run on a different thread use the same db connection as this test
```

swing and a miss :upside_down_face:

Reply:
Haha what an inexplicable mistake. I blame my mediocre German for thinking that "z" has a starting "t" already.